src/Module/Admin/Summary.php

```diff
@@ -26,7 +26,7 @@ public static function content()
 
 		// are there MyISAM tables in the DB? If so, trigger a warning message
 		$warningtext = [];
-		if (DBA::count('`information_schema`.`tables`', ['engine' => 'myisam', 'table_schema' => DBA::databaseName()])) {
+		if (DBA::count('information_schema`.`tables', ['engine' => 'myisam', 'table_schema' => DBA::databaseName()])) {
```

Comment:
This one doesn't seem right. Or is this a hack to count on system tables? If so, maybe there is a better solution directly in `DBA` possible?

Reply:
Like a optional flag in `count(..., $schema = null)` and if not null, use it

Reply:
In fact I just had a look at the code - it works that way but is ugly.

Reply:
I suggest that I apply some changes to the database class for this.

Reply:
I suggest doing it that way:
```
DBA::count(['information_schema' => 'tables'], ['engine' => 'myisam', 'table_schema' => DBA::databaseName()])
```

Reply:
much better of course! :-)

Reply:
Moved to #7174